cpp/include/raft/random/multi_variable_gaussian.cuh

```diff
@@ -59,6 +59,81 @@ class multi_variable_gaussian : public detail::multi_variable_gaussian_impl<T> {
   ~multi_variable_gaussian() { deinit(); }
 };  // end of multi_variable_gaussian
 
+/**
+ * @brief Matrix decomposition method for `compute_multi_variable_gaussian` to use.
+ *
+ * `compute_multi_variable_gaussian` can use any of the following methods.
+ *
+ * - `CHOLESKY`: Cholesky decomposition
+ * - `JACOBI`:
+
+ */
+using detail::multi_variable_gaussian_decomposition_method;
+using detail::multi_variable_gaussian_setup_token;
+
+template <typename ValueType>
+multi_variable_gaussian_setup_token<ValueType> setup_multi_variable_gaussian(
```

Comment:
Very minor nitpick: What do you think of calling this `build_multivariable_gaussian_token` or make_multi_variable_gaussian_token` just to be more consistent w/ other APIs? `setup` works as well, and doesn't require the use of `token` in the suffic so I'm okay w/ that name as well.

Reply:
I renamed this to `build_multi_variable_gaussian_token`; thanks!